git/cmd.py

```diff
@@ -794,7 +794,7 @@ def _kill_process(pid):
             else:
                 max_chunk_size = max_chunk_size if max_chunk_size and max_chunk_size > 0 else io.DEFAULT_BUFFER_SIZE
                 stream_copy(proc.stdout, output_stream, max_chunk_size)
-                stdout_value = output_stream
+                stdout_value = proc.stdout.read()
```

Comment:
That's probably the only way to do this properly, even though it seems funky.  Ideally `stdout_value` should be the value read, but in this case, it will always be an empty string as `read()` is called right after `stream_copy(...)`.
However, looking at it, the whole implementations seems to have gotten somewhat convoluted over the years, so it's probably alright after all.